htime/time.go

```diff
@@ -0,0 +1,68 @@
+package htime
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ONSdigital/dp-dd-hierarchy-importer/sql"
+)
+
+const (
+	en = "en"
+)
+
+var monthNames = map[int]string{
+	1:  "January",
+	2:  "February",
+	3:  "March",
+	4:  "April",
+	5:  "May",
+	6:  "June",
+	7:  "July",
+	8:  "August",
+	9:  "September",
+	10: "October",
+	11: "November",
+	12: "December",
+}
```

Comment:
Does this duplicate [time.Month String()](https://golang.org/pkg/time/#Month.String) method?